docs/cep/reference/functions/trigger/trigger.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cron Trigger
+---
+
+Triggers allow events to be periodically generated. Trigger definition can be used to define a trigger. A trigger also works like a stream with a predefined schema.
```

Comment:
Great intro! I suggest linking the word "stream" to the Streams section of the documentation.

Reply:
Is it "trigger" or "cron trigger"? The title does not agree with the text.